### PR TITLE
One week sprints

### PR DESCRIPTION
**Pull request by austin-schaefer**
Edits to agile handbook to reflect one-week sprints. 

I've requested three reviewers because I want something different from each of you:

* @homelessbirds : You always ask good questions and help ensure we say "why." Please take a look with a view to that.
* @paperclypse : Would this answer a new hire's questions? Does it describe your and my roles in this new weekly structure?
* @bradleycamacho : Does this accurately reflect how we've done one-week sprints in practice?

### CODE REVIEW

File: src/content/docs/agile-handbook/sprint-mechanics/liaisonships.mdx [hunk 1]
```diff
@@ -230,8 +230,4 @@ Not every project gets a liaisonship! For smaller projects, we'll encourage team
   >
     What is a hero? →
   </ButtonLink>
-</ButtonGroup>
-
-<Callout variant="tip">
-  We welcome thoughts or questions on our handbook! The easiest way to get in touch is to [file a GitHub issue](https://github.com/newrelic/docs-website/issues/new/choose).
-</Callout>
+</ButtonGroup>
```
Comment (paperclypse) [inline src/content/docs/agile-handbook/sprint-mechanics/liaisonships.mdx:233]: The liaisonship doc doesn't quite seem to line up with the current organizational model @austin-schaefer. But maybe that can wait until another time?

**Review by paperclypse — APPROVED** [review]
Looks good to me. I think these would be easy for a new writer to understand, or grok, if you will.

File: src/content/docs/agile-handbook/sprint-mechanics/meetings-and-ceremonies.mdx [hunk 2]
```diff
@@ -85,70 +57,63 @@ topics:
...
+Each Monday of a new sprint, we commit to a series of stories until we have filled our capacity for the sprint. Before the sprint planning meeting, the scrum master for each squad [calculates their point budget](/docs/agile-handbook/sprint-mechanics/planning-poker/#points_budget). 
 
-Secondly, all tickets should be groomed and ready on the Friday before sprint planning. This is because the goal is for all members of the squad to converge on the same estimation for each story which requires **everyone** to have a clear understanding of the scope of work. This means that not only do we need  good stories but we also need time and space to discuss these stories before sprint planning. 
+### Meeting rules [#meeting_rules]
 
-**The moral of the story: If we understand our own stories, then we can better explain them to our squad.** 
+* **Silent estimation:** At this point every story included in the sprint is well groomed and has been reviewed during grooming by all members of the squad. The entire squad reads each story silently. When all members of the squad have finished voting begins.
```
Comment (bradleycamacho) [inline src/content/docs/agile-handbook/sprint-mechanics/meetings-and-ceremonies.mdx:70]: We don't really do grooming anymore

File: src/content/docs/agile-handbook/sprint-mechanics/meetings-and-ceremonies.mdx [hunk 3]
```diff
@@ -85,70 +57,63 @@ topics:
...
 
-Secondly, all tickets should be groomed and ready on the Friday before sprint planning. This is because the goal is for all members of the squad to converge on the same estimation for each story which requires **everyone** to have a clear understanding of the scope of work. This means that not only do we need  good stories but we also need time and space to discuss these stories before sprint planning. 
+### Meeting rules [#meeting_rules]
 
-**The moral of the story: If we understand our own stories, then we can better explain them to our squad.** 
+* **Silent estimation:** At this point every story included in the sprint is well groomed and has been reviewed during grooming by all members of the squad. The entire squad reads each story silently. When all members of the squad have finished voting begins. 
+* **Blackout voting:**  In order to reduce influence, we blackout the cells in the spreadsheet until after a vote has happened. We change the background color to white to reveal the votes simultaneously.
```
Comment (bradleycamacho) [inline src/content/docs/agile-handbook/sprint-mechanics/meetings-and-ceremonies.mdx:71]: Might be nice to share a link to public version of the spreadsheet template if we can do that somehow. Not sure about security implications here

**Review by bradleycamacho — APPROVED** [review]
Just one small issue in regards to sprint grooming, but otherwise looks good!

File: src/content/docs/agile-handbook/sprint-mechanics/meetings-and-ceremonies.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/agile-handbook/sprint-mechanics/meetings-and-ceremonies.mdx:70]: But we may again at some point...?

File: src/content/docs/agile-handbook/sprint-mechanics/meetings-and-ceremonies.mdx (hunk 3 above)
Comment (paperclypse) [reply src/content/docs/agile-handbook/sprint-mechanics/meetings-and-ceremonies.mdx:71]: I don't see why we couldn't share an empty version of this template, especially if we restrict it to New Relic

Comment (bradleycamacho) [reply src/content/docs/agile-handbook/sprint-mechanics/meetings-and-ceremonies.mdx:71]: Oh, I was thinking making it public to everyone! We know these docs get a shocking amount of traffic, and I'd be surprised if all of it was from relics

**Pull request merged**